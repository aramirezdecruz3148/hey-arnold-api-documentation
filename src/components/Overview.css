.flex {
  display: flex;
  flex-direction: column;
  margin-left: 3vw;
}

.title {
  font-family: 'Luckiest Guy', cursive;
  color: #FFD90F;
  font-size: 3vw;
  margin-bottom: -.5vw;
  text-shadow:
		-.2vw -.2vw 0 #000,
		.2vw -.2vw 0 #000,
		-.2vw .2vw 0 #000,
    .2vw .2vw 0 #000;
}