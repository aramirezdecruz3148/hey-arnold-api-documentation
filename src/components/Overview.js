import React from 'react';

export default function Overview() {
  return (
    <>
      <h4>Overview</h4>
      <p>
        This is where I will explain the tech I used and the
        reason for creating the project.
      </p>
    </>
  );
}
