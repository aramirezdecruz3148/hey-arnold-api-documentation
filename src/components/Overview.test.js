import React from 'react';
import { shallow } from 'enzyme';
import Overview from './Overview';

describe('Overview component', () => {
  it('renders Overview', () => {
    const wrapper = shallow(<Overview />);
    expect(wrapper).toMatchSnapshot();
  });
});
