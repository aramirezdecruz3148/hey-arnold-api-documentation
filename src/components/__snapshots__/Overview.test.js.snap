// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview component renders Overview 1`] = `
<Fragment>
  <h4>
    Overview
  </h4>
  <p>
    This is where I will explain the tech I used and the reason for creating the project.
  </p>
</Fragment>
`;
